ttnn/cpp/ttnn/operations/eltwise/unary/unary_composite.hpp

```diff
@@ -285,26 +239,26 @@ constexpr auto normalize_hw = ttnn::register_operation_with_auto_launch_op<
 
 constexpr auto hardswish = ttnn::register_operation_with_auto_launch_op<
     "ttnn::hardswish",
-    operations::unary::ExecuteUnaryCompositeOpWithScaleShift<operations::unary::UnaryCompositeOpType::HARDSWISH>>();
+    operations::unary::ExecuteUnaryCompositeOpWithFloats<operations::unary::UnaryCompositeOpType::HARDSWISH>>();
 constexpr auto hardsigmoid = ttnn::register_operation_with_auto_launch_op<
     "ttnn::hardsigmoid",
-    operations::unary::ExecuteUnaryCompositeOpWithScaleShift<operations::unary::UnaryCompositeOpType::HARDSIGMOID>>();
+    operations::unary::ExecuteUnaryCompositeOpWithFloats<operations::unary::UnaryCompositeOpType::HARDSIGMOID>>();
 
 constexpr auto hardtanh = ttnn::register_operation_with_auto_launch_op<
     "ttnn::hardtanh",
-    operations::unary::ExecuteUnaryCompositeOpWithLowHigh<operations::unary::UnaryCompositeOpType::HARDTANH>>();
```

Comment:
I like these names better but will not block on this

Reply:
ok We try to change it from next time.